django/shortcuts.py

```diff
@@ -41,11 +43,19 @@ def redirect(to, *args, permanent=False, **kwargs):
         * A URL, which will be used as-is for the redirect location.
 
     Issues a temporary redirect by default; pass permanent=True to issue a
-    permanent redirect.
+    permanent redirect. Pass preserve_method=True to issue a redirect tha must
```

Comment:
I just spotted a minor typo in the docstring.

```suggestion
    permanent redirect. Pass preserve_method=True to issue a redirect that must
```